src/gui/mainwindow.cpp

```diff
@@ -1919,6 +1934,22 @@ void MainWindow::updatePowerManagementState()
 #if defined(Q_OS_WIN) || defined(Q_OS_MACOS)
 void MainWindow::checkProgramUpdate(const bool invokedByUser)
 {
+#ifdef Q_OS_WIN
+    if (!::IsWindows10OrGreater())
+    {
+        if (invokedByUser) {
+            auto *msgBox = new QMessageBox {QMessageBox::Information, u"qBittorrent"_qs
+                                           , tr("This is the last supported version for your Windows version.")
+                                           , QMessageBox::Ok, this};
+            msgBox->setAttribute(Qt::WA_DeleteOnClose);
+            msgBox->setWindowModality(Qt::NonModal);
```

Comment:
Why it is non-modal? In this case the user is available to produce multiple similar dialogs at a time.

Reply:
It's a copy-paste from here: https://github.com/qbittorrent/qBittorrent/blob/c8763f08da080cd9a9ea22caafff999ee54cbaa2/src/gui/mainwindow.cpp#L1778

Reply:
The non modal options was implemented with PR #16678.
On second read, that was due to the operation being asyncronous and the user might have opened other dialogs in the meantime.
I'll remove it.